Automattic/jetpack
# Stats: insert the stats column even when there is no comments col

Fixes #24481

#### Changes proposed in this Pull Request:

Some CPTs, when registered, do not declare that they support comments. We need to be able to add the Stats column even if a CPT does not have a comments column.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

1. Start from a site running Jetpack, with the Stats module active.
2. Now register a basic CPT by adding the following code snippet:

```php
add_action(
	'init',
	function () {
		register_post_type(
			'example_cpt',
			array(
				'label'               => __( 'Post Type', 'jeherve-cpt' ),
				'description'         => __( 'Post Type Description', 'jeherve-cpt' ),
				'labels'              => array(
					'name'          => _x( 'Post Types', 'Post Type General Name', 'jeherve-cpt' ),
					'singular_name' => _x( 'Post Type', 'Post Type Singular Name', 'jeherve-cpt' ),
				),
				'supports'            => array( 'title' ),
				'hierarchical'        => false,
				'public'              => true,
				'show_ui'             => true,
				'show_in_menu'        => true,
				'menu_position'       => 5,
				'show_in_admin_bar'   => true,
				'show_in_nav_menus'   => true,
				'can_export'          => true,
				'has_archive'         => true,
				'exclude_from_search' => false,
				'publicly_queryable'  => true,
				'capability_type'     => 'page',
			)
		);
	}
);
```
3. Go to the new Post Types menu; you should see a Stats column, just like in the Posts menu.
